modules/server/src/main/resources/db/migration/data/V1__add_protocols_table.sql

```diff
@@ -1,4 +1,6 @@
 CREATE TABLE protocols (
```

Comment:
Are you modifying an existing evolution?  I guess this is ok because the service is not deployed yet, right?

Reply:
Yes, I'm modifying current migrations because nobody is using this yet.